### PR TITLE
LHCb HEPDrone proposal

### PR DESCRIPTION
**Pull request by Tevien**
The fork contains a new project in association with LHCb with participation of CERN, NWO, and STFC

### CODE REVIEW

File: _gsocproposals/2018/proposal_LHCbHEPDrone.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Mentors
+
+ * [Sean Benson](mailto:sean.benson@cern.ch)
+ * [Konstantin Gizdov](mailto:konstantin.gizdov@cern.ch)
+
+
+## Links
+ * [HEPDrone arXiv](https://arxiv.org/abs/1712.09114)
+ * [source code](https://github.com/HEPDrone)
```
Comment (hegner) [inline _gsocproposals/2018/proposal_LHCbHEPDrone.md]: That link is invalid

File: _gsocproposals/2018/proposal_LHCbHEPDrone.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: Extension and characterisation of the LHCb data isolation drone neural networks
+layout: gsoc_proposal
+project: LHCb
+year: 2018
+organization: CERN, NWO, STFC
```
Comment (hegner) [inline _gsocproposals/2018/proposal_LHCbHEPDrone.md]: are you sure about STFC or don't you rather mean University of Edinburgh?

Comment (Tevien) [reply _gsocproposals/2018/proposal_LHCbHEPDrone.md]: STFC provide the funding for Konstantin, and NWO provides the funding for me. However, we are LHCb authors through the University of Edinburgh and Nikhef respectively. But on the other hand, NWO and STFC are the funding sources that are acknowledged in all LHCb papers as well as our reduced author list ones. So to bottom line it the institutions we go to are the University of Edinburgh and Nikhef but the organisations that provide the funding are the ones that I listed with the keyword.

Comment (Tevien) [reply _gsocproposals/2018/proposal_LHCbHEPDrone.md]: Just to add this is now changed to Nikhef and UoE.

File: _gsocproposals/2018/proposal_LHCbHEPDrone.md (hunk 1 above)
Comment (Tevien) [reply _gsocproposals/2018/proposal_LHCbHEPDrone.md]: Fixed, thanks for spotting.

File: _gsocproposals/2018/proposal_LHCbHEPDrone.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Extension and characterisation of the LHCb data isolation drone neural ne
 layout: gsoc_proposal
 project: LHCb
 year: 2018
-organization: CERN, NWO, STFC
+organization: CERN, Nikhef, Edinburgh
```
Comment (etejedor) [inline _gsocproposals/2018/proposal_LHCbHEPDrone.md]: Please use the list syntax to specify more than one organisation, as explained here:
http://hepsoftwarefoundation.org/gsoc/guideline.html

Comment (Tevien) [reply _gsocproposals/2018/proposal_LHCbHEPDrone.md]: This has been changed, thanks

**Comment by Tevien** [comment]
Thanks for the comments, I think all requested changes have been completed.

**Pull request merged**